tools/packaging/common/istio-iptables.sh

```diff
@@ -63,7 +63,7 @@ function dump {
 #  Function check if argument is a valid ipv4 or ipv6 address
 #
 function isValidIP() {
-   if [ "$1" != "${1#*[0-9].[0-9].[0-9].[0-9]}" ]; then
+   if [ "$1" != "${1#*[0-9]\.*[0-9]\.*[0-9]\.*[0-9]}" ]; then
```

Comment:
digging in more, I don't think this really fixes it.  The `*`s will glob any char.  IMO, moving to something like this is better:
```
if [[ ${ip_cidr} =~ ^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+\/?[0-9]*$ ]]; then 
  true 
elif ...
```

Reply:
Also, need to fix other places where it tries to strip the pattern--e.g. `isIPv4()`

I'm worried the ipv6 matching is buggy too